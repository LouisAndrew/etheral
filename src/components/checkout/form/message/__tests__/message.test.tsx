expect(true).toBe(true);

// import React from 'react';
// import ReactDOM from 'react-dom';
// import renderer from 'react-test-renderer';

// // import { render, cleanup } from '@testing-library/react'
// import { cleanup } from '@testing-library/react';
// import '@testing-library/jest-dom';

// import { Message } from '../message';

// describe('Message', () => {
//     const mockRegister = jest.fn(() => {});

//     const Element = <Message register={mockRegister} />;

//     afterEach(cleanup);

//     it('renders without crashing', () => {
//         const div = document.createElement('div');
//         ReactDOM.render(Element, div);
//     });

//     /* it('renders correctly', () => {
// 		const { getByTestId } = render()
// 	}) */

//     it('matches snapshot', () => {
//         const run = true;

//         if (run) {
//             const tree = renderer.create(Element).toJSON();
//             expect(tree).toMatchSnapshot();
//         }
//     });
// });
