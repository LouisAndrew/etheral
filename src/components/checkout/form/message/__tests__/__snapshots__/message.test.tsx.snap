// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message matches snapshot 1`] = `
<div
  className="css-vurnku"
>
  <h3
    className="css-r0fgiz"
  >
    PERSONALIZED MESSAGE (OPTIONAL)
  </h3>
  <div
    className="css-1ukulbb"
  >
    <div
      className="css-fwztci"
    >
      <label
        className="css-m85rj5"
        htmlFor="for-name"
      >
        FOR
      </label>
      <input
        className="css-4y926w"
        id="for-name"
        name="forName"
        placeholder="FOR"
        type="text"
      />
    </div>
    <div
      className="css-fwztci"
    >
      <label
        className="css-m85rj5"
        htmlFor="from-name"
      >
        FROM
      </label>
      <input
        className="css-4y926w"
        id="from-name"
        name="fromName"
        placeholder="FROM"
        type="text"
      />
    </div>
    <div
      className="css-16vu25q"
    >
      <label
        className="css-knfhfe"
        htmlFor="message"
      >
        MESSAGE (
        100
         remaining)
      </label>
      <textarea
        className="custom-scrollbar css-7ehb20"
        id="message"
        maxLength={100}
        name="message"
        onChange={[Function]}
        placeholder="INSERT YOUR MESSAGE (MAX 100 CHAR)"
      />
    </div>
  </div>
</div>
`;
